static/app/views/preprod/utils.tsx

```diff
@@ -0,0 +1,51 @@
+import {BuildDetailsArtifactType, type Platform} from 'sentry/views/preprod/types';
+
+export function formatBytes(bytes: number, isSi = false) {
+  if (bytes === 0) return '0 B';
+  const k = isSi ? 1000 : 1024;
+  const sizes = isSi ? ['B', 'KB', 'MB', 'GB', 'TB'] : ['B', 'KiB', 'MiB', 'GiB', 'TiB'];
+  const i = Math.floor(Math.log(bytes) / Math.log(k));
+  return parseFloat((bytes / Math.pow(k, i)).toFixed(2)) + ' ' + sizes[i];
+}
```

Comment:
I'm 100% sure we implement this somewhere already. We want to port this over to a common formatters util in our design system so we don't keep reinventing the wheel